### ISSUE TITLE
Angular 12

### ISSUE THREAD

**Issue by mrahhal** [ui-router/angular#919]
This is not compatible with angular 12 right now because of the peer dependencies versions.

**Comment by sebastian-zarzycki-es** [comment]
Hopefully only a ver bump?

**Comment by PikaRules** [comment]
when is the support for version 12 going to be released?

**Comment by sebastian-zarzycki-es** [comment]
I'm also waiting (im)patiently :)

**Comment by dpraul** [comment]
FWIW, the current version appears to work totally fine for me with Angular 12 (ignoring the peer dependency issues). I haven't put it through rigorous testing quite yet but it's working from what I've seen.

**Comment by ngehlert** [comment]
well according to https://github.com/ui-router/angular/issues/934 it's not that tivial maybe? 

>  I would be happy to merge a PR supporting Angular 12 but this does involve some no-obvious steps such as updating the angular-hybrid modules, and bumping the Angular version of the sample apps, etc.

**Comment by dpraul** [comment]
We've had ui-router running on Angular 12 in production for a couple weeks now with no issues. We still get the peer-dependency warning at install-time, but it appears to work just fine.

**Comment by stnor** [comment]
TL;DR: I would very much appreciate a release for Angular 12 too.

Ui-router for the hybrid app and Angular has been absolutely critical for us, since we've migrated a very large angularjs app to angular 10 over the course of the last 12 months. We are still running ui-router in the non-hybrid, and have no plans as of current to migrate to the Angular.

**Comment by christopherthielen** [comment]
uirouter/angular v9.1.x supports ng12 and ng13
